exercises/twelve-days/src/main/java/TwelveDays.java

```diff
@@ -0,0 +1,9 @@
+public class TwelveDays {
+    public String verse(int verseNumber) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    public String verses(int startVerse, int endVerse) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+}
```

Comment:
Please remove the `public` keyword from the class and method signatures here; last change request, promise!